doc/sphinx/source/community/detailed_release_procedure.rst

```diff
@@ -0,0 +1,190 @@
+  On DKRZ/Levante, a user can't have more than 20 SLURM jobs running at a time.
+  As soon as a job is finished, the next one should start
+
+Submit the batch scripts that will run all recipes. Assemble some statistics so that issues with certain recipes
+can be followed-up, and document this information in the release issue, examples:
+
+- number of successfully run recipes
+- number of failed recipes with Diagnostic error (can they be fixed? Can the fixes be included in the release?)
+- number of recipes that are missing data
+- number of recipes that have various other issues (and document them)
+
+To parse the output of all these runs use the `parse_recipes_output.py` Python script, included at the
+same locations where the generation script is.
+
+Running the comparison
+----------------------
+
+To compare the newly produced output from running all recipes, follow these steps below.
+
+Login and access to the DKRZ esmvaltool virtual machine (VM) - results from recipe runs
+are stored on the VM; login with:
+
+.. code-block:: bash
+
+  ssh user@esmvaltool.dkrz.de
+
+where `user` is your DKRZ/Levante user name; then get and install miniconda on the VM, and
+if you have a Miniconda installer already downloaded in your Levante $HOME
+
+.. code-block:: bash
+
+  scp Miniconda3-py39_4.12.0-Linux-x86_64.sh user@esmvaltool.dkrz.de:~
+
+Next, we need to set up the input files
+
+.. note::
+
+  If you wrote recipe runs output to Levante's `/scratch` partition, be aware that
+  the data will be removed after two weeks, so you will have to move the output data
+  to the `/work` partition, via e.g. a `nohup` job:
+
+  .. code-block:: bash
+
+    nohup cp -r /scratch/b/$USER/esmvaltool_output/* /work/bd0854/b382109/v2xx
+
+  where `bd0854/b382109` is the project location in `work`
+
+
+The `/work` partition is visible by the VM so you can run the compare tool straight on the VM.
+
+Do not store final release results on the VM including `/preproc/` dirs, the total
+size for all the recipes output, including `/preproc/` dirs is in the 4.5TB ballpark,
+much too high for the VM storage capacity!
+
+The steps to running the compare tool on the VM are the following:
+
+- run date: log the run date here
+- conda env: log the name of the conda environment you are using
+- ESMValTool branch: log the name of the code branch you are using (e.g. `v2.8.x`)
+- prerquisite - install `imagehash`: `pip install imagehash`
+- reference run (v2.7.0): `export reference_dir=/work/bd0854/b382109/v270` (contains `preproc/` dirs too, 122 recipes)
+- current run (v2.8.0): `export current_dir=path_to_current_run`
+- command to run: `nohup python ESMValTool/esmvaltool/utils/testing/regression/compare.py --reference $reference_dir --current $current_dir > compare_v280_output.txt`
+
+Some of the recipes will appear as having identical output to the one from previous release. However, others
+will need human inspection; here are some guidelines on how to perform the human inspection:
```

Comment:
Should the human inspection be done by the release manager? I would try to ask recipe maintainers for this.

Reply:
err yes and no - that would delay the process, but it'd also accelerate it if peeps were on point - let's talk about it

Reply:
I added some information to ask maintainers and the development community to support the RM